### DISCUSSION TITLE
How to add a shadow to a badge when hovered?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#8925] [General - Components / Directives / etc] [answered]
I would like to add a [`shadow-14` class](https://quasar.dev/style/shadows#usage) to a badge when the mouse passes over it (hover). 

Is there a "the cursor is currently above me" flag so that I can use a dynamic class?

**Comment by rstoenescu** [comment, marked as answer]
':hover' will do the job

**Reply by wsw70** [reply]
I just tried to add

    :hover="hoverAboveTag"

in a `q-badge` definition, and then

```
methods: {
    hoverAboveTag: (x) => console.log(`hover: ${x}`), // hopefully catch "hover in" and "hover out"
...
```
but nothing happened when hovering above the badge.

**Reply by metalsadman** [reply]
`:hover` as a pseudo-class in css, but if you want it as an event then use `@hover`.

**Reply by wsw70** [reply]
Ah, I should have realized by myself that I should have watched the emitted events and catch them accordingly, thanks.

In the case of `q-badge` there is no event on hover but I found out that `q-chip` will be better as it handles the "show that we are being hovered on" by default.
